pkg/operator/ceph/object/bucket.go

```diff
@@ -140,7 +147,12 @@ func getBucketMetadata(c *Context, bucket string) (*ObjectBucketMetadata, bool,
 		return nil, false, errors.Wrapf(err, "failed to read buckets list result=%s", result)
 	}
 
-	createdAt, err := time.Parse("2006-01-02 15:04:05.999999999Z", s.Data.CreationTime)
+	timeParser := beforeOctopusTime
+	cephOperatorBaseImageVersion := os.Getenv(operatorCephBaseImageVersion)
+	if strings.Contains(cephOperatorBaseImageVersion, "octopus") {
```

Comment:
Do we need to check for "not nautilus" so that pacific and future versions will also use the new format?

Reply:
Oopps! New version